components/Receiver.js

```diff
@@ -26,7 +27,7 @@ class Receiver extends Component {
           />
         <ScrollView snapToAlignment="center" decelerationRate="fast" horizontal >
           {this.state.timers.map((task) => {
-            return <View style={styles.task}>
+            return <View style={styles.task} key={task.id}>
```

Comment:
Nice! I think this gets rid of some of the errors. I also found a "key extractor", although this might be in reference to solely the imported <FlatList /> component. I think it's on line 50 of CareTaker and looks like 
```
keyExtractor={(item) => item.id.toString()}
```
just a mention for two different ways we can tackle this.